hide-ignored/hide-ignored-flags.js

```diff
@@ -1,31 +1,66 @@
 // ==UserScript==
-// @name         Hide the Ignored
-// @homepage     https://gist.github.com/Bhargav-Rao/fd4202b609b9610f2aeb3df350513be7
-// @version      0.2
-// @description  Hides those posts that you've already seen in the queue
+// @name         Hide the Visited
```

Comment:
Any reason to change the name?

Reply:
The system calls them 'visited posts' (see the CSS selector). But perhaps the script can be extended with an option to mark individual posts as ignored (in local storage), so I'll change it back.